browser/request_otr/request_otr_browsertest.cc

```diff
@@ -466,26 +466,6 @@ class RequestOTRServiceWorkerBrowserTest : public RequestOTRBrowserTest {
   RequestOTRServiceWorkerBrowserTest()
       : https_server_(net::EmbeddedTestServer::TYPE_HTTPS) {}
 
-  void SetUp() override {
-    // We still need this so InstallMockExtension() can find its files.
-    brave::RegisterPathProvider();
-
-    // Reuse upstream test files in content.
-    content::RegisterPathProvider();
```

Comment:
was this weird behavior (registering our path provider... then content path provider) needed because paths were registered so many times? Really curious why that was added 🤔

Reply:
all of the path registration stuff was a mess. It's fixed now